### PR TITLE
Kelsha Darby -  User Story 38 & 40

### LINKED ISSUE #41: User Story 38, Admin disables a merchant account
As an admin
When I visit the admin's merchant index page ('/admin/merchants')
I see a "disable" button next to any merchants who are not yet disabled
When I click on the "disable" button
I am returned to the admin's merchant index page where I see that the merchant's account is now disabled
And I see a flash message that the merchant's account is now disabled

### LINKED ISSUE #43: User Story 40, Admin enables a merchant account
As an admin
When I visit the merchant index page
I see an "enable" button next to any merchants whose accounts are disabled
When I click on the "enable" button
I am returned to the admin's merchant index page where I see that the merchant's account is now enabled
And I see a flash message that the merchant's account is now enabled

### PR DESCRIPTION
**Pull request by kelshadarby**
## User Story 38 - An Admin Can Disable a Merchant's Account
Closes #41 

**_Test_**
- '/spec/features/admin/merchant_index_spec'
- '/spec/models/merchant_spec'

**_Database_**
- Add boolean column to merchants table to show if a merchant is active or inactive

**_Routes_**
- Added admin merchant index route
- Added admin merchant disable route

**_Controller_**
- Add disable method that does the following:
    - Finds a merchant using params
    - Disables that merchant using a disable model method
    - Shows a flash message saying that the merchant has been disabled
    - Redirects to the admin merchants index page

**_Model_**
- `disabled?`
    - This method checks if the merchant is active or not
- `disable`
    - This method updates the merchant's active status to false

**_View_**
- Create admin merchant index view
     - This view shows a button next to the merchants name is they are not disabled, otherwise it shows "Disabled"

_____
## User Story 40 - An Admin Can Enable a Merchant's Account
Closes #43 

**_Test_**
- Write test to check that an admin can enable a merchants account

**_Routes_**
- Edit route in user story 38 to go to an update method rather than a specific disable method

**_View_**
- Add conditional to view to put an enable button if the merchant is inactive

**_Controller_**
- Add conditional to disable a merchant if they are active and disable them if they aren't

### CODE REVIEW

File: app/controllers/admin/merchants_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Admin::MerchantsController < Admin::BaseController
+  def index
+    @merchants = Merchant.all
+  end
+
+  def update
+    merchant = Merchant.find(params[:merchant_id])
+    if merchant.active?
+      merchant.disable
+      flash[:disabled] = "#{merchant.name} has been disabled."
```
Comment (fredrondina) [inline app/controllers/admin/merchants_controller.rb:10]: nice job having update activate and disable in one method

File: db/migrate/20200413034746_add_column_to_merchants.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddColumnToMerchants < ActiveRecord::Migration[5.1]
+  def change
+    add_column :merchants, :active?, :boolean, default: true
```
Comment (fredrondina) [inline db/migrate/20200413034746_add_column_to_merchants.rb:3]: nice migration

**Review by fredrondina — APPROVED** [review]
Fantastic work! Super glad you were able to get this in!

File: app/controllers/admin/merchants_controller.rb (hunk 1 above)
Comment (brian-greeson) [inline app/controllers/admin/merchants_controller.rb:10]: I like the use of helper methods here

**Review by brian-greeson — APPROVED** [review]
Looks Great!

**Pull request merged**